README.md

```diff
@@ -46,12 +60,14 @@ KSMT can express formulas in the following theories:
 | Uninterpreted Functions | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 | Arithmetic              | :heavy_check_mark: |                    | :heavy_check_mark: | :heavy_check_mark: |
 
-Check out our [roadmap](Requirements.md) for detailed description of features and future plans.
+### Kotlin-based DSL for SMT formulas
+
+Various scenarios are available for using SMT solvers: you can use a single solver to determine whether a formula is
+satisfiable, or you can apply several solvers to the same expression successively (i.e., use solvers in a _joint_
```

Comment:
`joint mode` is not a common term. Perhaps we can drop this clarification in parentheses.